src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,90 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+
 public class WorkWithFile {
+    private static final String DESIRED_LETTER = "w";
+
+    public String[] readFromFile(String fileName) {
```

Comment:
In this task, you should have only one public method -- getStatistics.
Which should use private decomposed methods.

```
String data = readFromFile(fromFileName);
String report = createReport(data);
writeToFile(report, toFileName);
```